waiter/bin/ci/run-integration-tests.sh

```diff
@@ -40,20 +43,27 @@ if [ ${MINIMESOS_EXIT_CODE} -ne 0 ]; then
     echo "minimesos failed to startup -- exiting"
     exit ${MINIMESOS_EXIT_CODE}
 fi
+$(minimesos info | grep MINIMESOS)
 
 # Start waiter
-${WAITER_DIR}/bin/run-using-minimesos.sh 9091 &
+WAITER_PORT=9091
+${WAITER_DIR}/bin/run-using-minimesos.sh ${WAITER_PORT} &
 
 # Wait for waiter to be listening
-timeout 180s bash -c "wait_for_waiter 9091"
+timeout 180s bash -c "wait_for_waiter ${WAITER_PORT}"
 if [ $? -ne 0 ]; then
   echo "$(date +%H:%M:%S) timed out waiting for waiter to start listening, displaying waiter log"
   cat ${WAITER_DIR}/log/waiter.log
   exit 1
 fi
 
+# Set WAITER_URI, which is used by the integration tests
+export WAITER_URI=localhost:${WAITER_PORT}
```

Comment:
Should we use `127.0.0.1` instead of `lcoalhost` (https://www.reddit.com/r/programming/comments/6g8zc8/how_changing_localhost_to_127001_sped_up_my_test/)